api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -173,6 +178,9 @@ message ExternalProcessor {
   // :ref:`clear_route_cache <envoy_v3_api_field_service.ext_proc.v3.CommonResponse.clear_route_cache>`
   // field is set in an external processor response.
   bool disable_clear_route_cache = 11;
+
+  // Additional metadata to be added to the dynamic filter state for logging purposes.
```

Comment:
Discussed offline with @rshriram.
This may be better off in the (dynamic) metadata rather than in the filter-state.

Reply:
Based on the followup discussion, do you still feel that we need to store the metadata into dynamic metadata while storing cluster metadata into filter state?

Reply:
I think filter state is fine here, but I do question why it's so inconsistent across ext_proc and fault injection.

Reply:
I do have a question though, why is this `Metadata`, which is really a per-filter construct thing and not just a general purpose Struct? I think `Metadata` makes sense if there is a naturally filer-based namespace, but not so much here.